website/sidebars.js

```diff
@@ -103,17 +103,6 @@ module.exports = {
   "dbt CLI": [
     "dbt-cli/cli-overview",
     "dbt-cli/installation",
-    {
-      type: "category",
-      label: "Installation guides",
-      items: [
-        "dbt-cli/installation-guides/macos",
-        "dbt-cli/installation-guides/windows",
-        "dbt-cli/installation-guides/ubuntu-debian",
-        "dbt-cli/installation-guides/centos",
-        "dbt-cli/installation-guides/install-from-source",
```

Comment:
Please add these to the `_redirects` file

Reply:
I may have totally misunderstood the meaning of that phrase, but I believe I already have. 🤔 when I search in `_redirects`, each one is present.

Reply:
Hmmm, so I think we need a new entry:

For `windows` we currently have a redirect from when we migrated from readme.io to docs.getdbt.com:
```
/docs/windows	/dbt-cli/installation-guides/windows	302
```

Now what we need is to update the existing entry _and_ add a new entry:
```
/docs/windows	/dbt-cli/installation	302
/dbt-cli/installation-guides/windows	/dbt-cli/installation	302
```

If you want to tinker, I'd love a little make recipe where I can do something like:
```
# make update_redirects from_path to_path
$ make update_redirects /dbt-cli/installation-guides/windows /dbt-cli/installation
```
Which would:
- search the _redirects file for any entries that have the `from_path` as the target, and replaces it with the `to_path`
- add a new entry to the _redirects file for the new mapping

Just an idea :bulb:

Reply:
Added an issue for the make tool and added these accordingly.